components/CocktailsOverview/CocktailsOverview.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/jsx-key */
+import Image from "next/image";
+import useSWR from "swr";
+import styled from "styled-components";
+
+const URL = "https://www.thecocktaildb.com/api/json/v1/1/search.php?s=cocktail";
+
+export default function CocktailsOverview({}) {
```

Comment:
```suggestion
export default function CocktailsOverview() {
```
You can remove these curly brackets if you do not destructure anything